ffwd-http-reporter/src/test/java/com/spotify/metrics/ffwd/FastForwardHttpReporterTest.java

```diff
@@ -83,63 +110,68 @@ public Double getValue() {
                 }
             });
 
+        setupDistribution();
+
         final Set<Batch.Point> expected = new HashSet<>();
+        expected.add(new Batch.Point("prefix.distribution",
+                of( "unit", "n", "stat", "distribution","metric_type",
+                        "distribution"), RESOURCE, Value.DistributionValue.create(VALUE_1), TIME));
         expected.add(new Batch.Point("prefix.counter",
-            of("unit", "n", "stat", "count", "metric_type", "counter"), 0, TIME));
+            of("unit", "n", "stat", "count", "metric_type", "counter"), RESOURCE, VALUE_0, TIME));
         expected.add(new Batch.Point("prefix.deriving-meter",
-            of("unit", "n/s", "stat", "5m", "metric_type", "deriving-meter"), 0, TIME));
+            of("unit", "n/s", "stat", "5m", "metric_type", "deriving-meter"), RESOURCE, VALUE_0, TIME));
         expected.add(new Batch.Point("prefix.deriving-meter",
-            of("unit", "n/s", "stat", "1m", "metric_type", "deriving-meter"), 0, TIME));
+            of("unit", "n/s", "stat", "1m", "metric_type", "deriving-meter"), RESOURCE, VALUE_0, TIME));
         expected.add(new Batch.Point("prefix.histogram",
-            of("unit", "n", "stat", "max", "metric_type", "histogram"), 0, TIME));
+            of("unit", "n", "stat", "max", "metric_type", "histogram"), RESOURCE, VALUE_0, TIME));
         expected.add(new Batch.Point("prefix.histogram",
-            of("unit", "n", "stat", "min", "metric_type", "histogram"), 0, TIME));
+            of("unit", "n", "stat", "min", "metric_type", "histogram"), RESOURCE, VALUE_0, TIME));
```

Comment:
is batch point - key, tags, resource, value, and then timestamp? so you'd add a resource value but still need that 0 afterwards, right?
Similar to above: 
points.add(new Batch.Point(key, statsMap(stat), createResource(),
                    Value.DoubleValue.create(value), timestamp));

Reply:
Yes, now it is => key, tags, resource, value, and then timestamp. 
 VALUE_0 = Value.DoubleValue.create(0)
Resource is always empty. I hope that answer your question

Reply:
ah i was reading the , as _. this is what happens when i review in the morning lol.